Added immutable standard property (#21)
ivoa-std/VOSpace: VOSpace.tex

    \item .null indicates an endpoint that discards all data written to it and provides no data when read from, i.e., a bit bucket
\end{itemize}

\subsubsection{Immutable nodes}

> \label{subsubsec:immutable nodes}

Reviewer: Spaces in a label probably work, but why take the chance?
```suggestion
\label{subsubsec:immutable-nodes}
```

Author: I used the space because I noticed that it is used in other parts of the documents (e.g. `\label{subsec:nodes and node types}` and `\label{sec:vospace data model}`), but I agree that using a hyphen is better.

Author: It seems that all the labels of the document contain spaces.

Reviewer: I hadn't seen that, serves me right for not looking. The references to section labels with spaces seem to work fine, so not an issue then.

> A VOSpace service can support, as an OPTIONAL feature, immutable nodes. Content of immutable nodes can't be changed and they are identified by adding the standard property ivo://ivoa.net/vospace/core\#immutable set to true.

Reviewer: Just a suggestion.
```suggestion
A VOSpace service MAY support immutable nodes. Content of immutable nodes can't be changed and they are identified by adding the standard property ivo://ivoa.net/vospace/core\#immutable set to true.
```

Author: Done.

    \item ivo://ivoa.net/vospace/core\#mtime SHALL be used as the property URI denoting the data modification time
    \item ivo://ivoa.net/vospace/core\#ctime SHALL be used as the property URI denoting status change (aka metadata modification) time
    \item ivo://ivoa.net/vospace/core\#btime SHALL be used as the property URI denoting initial creation time

>     \item ivo://ivoa.net/vospace/core\#immutable MAY be used as the property URI denoting a node whose content cannot be changed and for which moveNode and deleteNode operations are not allowed. If the node is a container node, child nodes cannot be added or removed. Children of an immutable node inherit this property. Properties that are not readOnly can still be edited

Reviewer: Move the details of what `immutable` means to a separate section, giving us more room to describe it in detail.
```suggestion
    \item ivo://ivoa.net/vospace/core\#immutable MAY be used as the property URI denoting a node whose content cannot be changed and for which moveNode and deleteNode operations are not allowed. See section \ref{tbd} for details.
```

Author: I created a new section. I put it under "Nodes and node types" section, since it describes a special nodes behavior. Let me know if you prefer to move it somewhere else, for example at the end of the properties section. I added also a condition for link nodes (target can't be changed), but I'm not sure if it should be possible to change the target using the setNode operation.

Reviewer: Looks good. :+1:

Reviewer: > I'm not sure if it should be possible to change the target using the setNode operation.

The target is effectively the content for a LinkNode, so it once the node is made immutable it you shouldn't be able to direct the link to point to something else.
